components/brand/Features.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <section class="container mx-auto px-6 md:px-8 mt-[83px]">
+    <h3
+      class="uppercase text-primary text-[32px] md:text-5xl text-center font-semibold leading-tight mb-16"
+    >
+      PRODUCT FEATURES
+    </h3>
+    <div class="flex flex-wrap-reverse xl:flex-nowrap items-center justify-center justify-between mb-12 lg:mb-24">
```

Comment:
```suggestion
    <div class="flex flex-wrap-reverse xl:flex-nowrap items-center justify-center xl:justify-between mb-12 lg:mb-24">
```

`justify-center` was being overridden by `justify-between`, so `justify-center` had no effect here.

However, it will be good to have `justify-center` for the mobile view. Since for screens wider than 1280px we are showing the two columns but only one column for narrower screens, it will be good to center that content for these narrow screens.